common/taskana-common-logging/src/test/java/outside/of/pro/staskana/OutsideOfProTaskanaPackageLoggingTestClass.java

```diff
@@ -0,0 +1,7 @@
+package outside.of.pro.staskana;
```

Comment:
staskana? seems like we have a new name for a spin off :D

Reply:
Oops.. 
That was part of my negative testing and I forgot to rename the package again €.< 

Well. Something I'll keep in mind for the master PR